src/utils/interfaces.ts

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-import { ClickMode, HoverMode, InteractivityDetect, MoveDirection, OutMode, ShapeType } from "./enums";
+import { ClickMode, HoverMode, OutMode } from './enums';
+import { InteractivityDetect, MoveDirection, ShapeType, ProcessBubbleType } from "./enums";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multiple imports from './enums' can be combined into one.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5012291)